qa/acceptance/spec/shared_examples/cli/logstash-plugin/update.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+
+shared_examples "logstash update" do |logstash|
+  before :each do
+    logstash.install(LOGSTASH_VERSION)
+  end
+
+  after :each do
+    logstash.uninstall
+  end
+
+  let(:plugin_name) { "logstash-filter-qatest" }
+  let(:previous_version) { "0.1.0" }
+
+  before do
+    logstash.run_command_in_path("bin/logstash-plugin install --version #{previous_version} #{plugin_name}")
+    # Logstash wont update when we have a pinned versionin the gemfile so we remove them
```

Comment:
At first I though it was a bug than I did a quick chat with @colinsurprenant about and I think the conclusion was that we need some kind of flag to allow override of pinned version. I will create an issue to discuss it.